apache/superset
# feat: Adds the ECharts Bubble chart

### SUMMARY
Adds the ECharts Bubble chart.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
1. Chart and data control panel:
![Screenshot from 2022-11-23 10-52-20](https://user-images.githubusercontent.com/12967587/203477116-08c4fa72-daac-4465-9290-2511eac70139.png)

2. Customize control panel
![Screenshot from 2022-11-27 16-18-56](https://user-images.githubusercontent.com/12967587/204131304-8b142e5e-acd2-436f-acb9-7d3f2aa66f94.png)

3. Tooltip:
![Screenshot from 2022-11-23 11-11-23](https://user-images.githubusercontent.com/12967587/203477703-60049948-5e74-47e7-a06f-b7180f323a8b.png)

### TESTING INSTRUCTIONS
1. create d3 bubble chart
2. merge commits from this PR and open existing chart should work fine as before.
4. test all controls in `data` and `customize` sections.

### ADDITIONAL INFORMATION
